pkg/controllers/vdb/revivedb_reconciler.go

```diff
@@ -243,7 +243,7 @@ func (r *ReviveDBReconciler) getPodList() ([]*PodFact, bool) {
 // findPodToRunInit will return a PodFact of the pod that should run the init
 // command from
 func (r *ReviveDBReconciler) findPodToRunInit() (*PodFact, bool) {
-	return r.PFacts.findPodToRunAdminCmdOffline()
+	return r.PFacts.findPodToRunAdminCmdOffline("" /* sandbox name*/)
```

Comment:
I think this API call will change, but we should introduce a constant called `MainCluster`, which is an empty string.